components/ibm-components/ffdl/serve/component.yaml

```diff
@@ -0,0 +1,37 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: 'Seldon Core - Serve Model'
+description: |
+  Serve Machine Learning and Deep Learning Models remotely as web service using Seldon Core
+inputs:
+  - {name: model_id,            description: 'Required. Model training_id from Fabric for Deep Learning'}
+  - {name: deployment_name,     description: 'Required. Deployment name for the seldon service'}
+  - {name: model_class_name,    description: 'PyTorch model class name', default: 'ModelClass'}
```

Comment:
Is this specific to PyTorch models? Should we call this 'Seldon Core - Serve PyTorch Model' ? Or make it mroe generic?

Reply:
Thanks for the review. The component container itself could be more generic, but maybe this component.yaml implementation is more specific for PyTorch model. Let me rename the description.

Reply:
done